Toggle dark/light mode manually (#129)
vadimdemedes/tailwind-rn: readme.md

import utilities from './tailwind.json';

const App = () => (

> 	<TailwindProvider utilities={utilities} theme="dark">

Reviewer: This example isn't related to the introduction of the new prop, could you undo the change here?

		<MyComponent />
	</TailwindProvider>
);
```

> #### theme

Reviewer: Could you rename the prop to `colorScheme` to match the naming in the codebase?

> Type: `string | undefined | null`

Reviewer: ```suggestion
Type: `string`
```

> Manually set theme mode: `light` or `dark`. This prop will override the native color scheme and its an optional prop.

Reviewer: ```suggestion
Override the default color scheme. Possible values are `light` or `dark`.
```
